config/routes.rb

```diff
@@ -7,4 +7,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "users", to: "users#index"
+  get "/users/:id", to: "users#show"
+  get "/users/:id/posts", to: "users#post"
+  get "/users/:id/posts/:idp", to: "users#detail"
```

Comment:
- [ ] these routes should be for `"post#index"` and `"post#show"` kindly fix this and also fix the view folder

Reply:
fixed.
![image](https://user-images.githubusercontent.com/13853450/192113145-a49a14a5-5680-4ab9-98f9-f41516af8c6f.png)